third_party/terraform/tests/resource_logging_billing_account_sink_test.go

```diff
@@ -221,6 +279,36 @@ resource "google_storage_bucket" "log-bucket" {
 `, name, billingAccount, getTestProjectFromEnv(), bucketName)
 }
 
+func testAccLoggingBillingAccountSink_described(name, bucketName, billingAccount string) string {
+	return fmt.Sprintf(`
+resource "google_logging_billing_account_sink" "described" {
+  name            = "%s"
+  billing_account = "%s"
+  destination     = "storage.googleapis.com/${google_storage_bucket.log-bucket.name}"
+  filter          = "logName=\"projects/%s/logs/compute.googleapis.com%%2Factivity_log\" AND severity>=ERROR"
+}
+
+resource "google_storage_bucket" "log-bucket" {
+  name = "%s"
+}
+`, name, billingAccount, getTestProjectFromEnv(), bucketName)
+}
+
+func testAccLoggingBillingAccountSink_disabled(name, bucketName, billingAccount string) string {
```

Comment:
Similar, `disabled` was left out of this config

Reply:
done